packages/api/utils/ipLogging.ts

```diff
@@ -0,0 +1,14 @@
+import middy from "@middy/core";
+import { APIGatewayProxyEvent, APIGatewayProxyResult } from "aws-lambda";
+
+// Create the middleware and export it
+export const IpCheckerMiddleware = (): middy.MiddlewareObj<APIGatewayProxyEvent, APIGatewayProxyResult> => {
+  // Set up a before check, this will run before the handler
+  const before: middy.MiddlewareFn<APIGatewayProxyEvent, APIGatewayProxyResult> = async (request): Promise<void> => {
+    // log the sourceIp
+    console.log(request.event.requestContext.identity.sourceIp);
```

Comment:
Can you add context to the line here so that it doesn't just come across as `1.2.3.4` but also tells us what is being logged?

Reply:
Suggest something like
`console.log("Source IP:", request.event.requestContext.identity.sourceIp);`

Reply:
```suggestion
    console.log("Client IP address: " + request.event.requestContext.identity.sourceIp);
```